unicodetools/src/test/java/org/unicode/propstest/CheckGCB.java

```diff
@@ -33,283 +33,283 @@
     private static final IndexUnicodeProperties IUP = IndexUnicodeProperties.make(Settings.latestVersion);
 
     private static final Set<PropertyNames.Named> TO_SKIP = ImmutableSet.of(
-	    Line_Break_Values.Complex_Context,
+        Line_Break_Values.Complex_Context,
 
-	    Word_Break_Values.Regional_Indicator
-	    );
+        Word_Break_Values.Regional_Indicator
+        );
 
     public static void main(String[] args) {
         final Segmenter gcb =
                 Segmenter.make(IUP, "GraphemeClusterBreak", Segmenter.Target.FOR_UCD).make();
-	final UnicodeMap<UcdPropertyValues.Grapheme_Cluster_Break_Values> gcbProps = IUP.loadEnum(UcdProperty.Grapheme_Cluster_Break);
+    final UnicodeMap<UcdPropertyValues.Grapheme_Cluster_Break_Values> gcbProps = IUP.loadEnum(UcdProperty.Grapheme_Cluster_Break);
```

Comment:
This file wanted 8sp/tab.

Reply:
Fixed in https://github.com/unicode-org/unicodetools/pull/178/commits/7c5c10bdc0232617b772ea6dd0e1e9389a3a9cfc